src/ui/pages/NotificationDetails/components/ReceiveCredential/ReceiveCredential.tsx

```diff
@@ -66,7 +65,7 @@ const ReceiveCredential = ({
       setTimeout(() => {
```

Comment:
@sdisalvo-crd The reason we added a timeout was in case `acceptAcdc` is very fast and the animation does not complete. The correct approach would be to save the timestamp at the start of the function, and have `setTimeout` with `ANIMATION_DELAY - (timeElapsed)`

Reply:
Thanks @iFergal, I couldn't really remember why we had this delay in the animation, I thought it was left from before. I'll try your suggestions, thanks for jumping in 👍

Reply:
@sdisalvo-crd I believe Duke already fixed it in another PR

Reply:
It's not really a delay, more so it was something to make sure the animation has fully played irregardless of how long it takes to call the core. But it wasn't done correctly the first time - it should be now

Reply:
Oh thanks, I was already opening another ticket for it. I will double check that with Duke. Thank you 🙏 .